### PR TITLE
[FIX] website_[sale,delivery]: update freeshipping on delivery change

### PR DESCRIPTION
**Pull request by kebeclibre**
Since those two modules do not have a bridge, there was no support for changing the freeshipping line
when changing the delivery carrier

Now it has, at the price of ugly code

OPW 1871526

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by kebeclibre** [comment]
@JKE-be 
@nim-odoo

**Comment by nim-odoo** [comment]
will do ASAP

**Comment by xmo-odoo** [comment]
ping

File: addons/website_sale_delivery/static/src/js/website_sale_delivery.js [hunk 1]
```diff
@@ -14,6 +16,17 @@ odoo.define('website_sale_delivery.checkout', function (require) {
...
         var $compute_badge = $('#delivery_carrier input[name="delivery_type"][value=' + result.carrier_id + '] ~ .o_delivery_compute');
+        var $discount = $('#order_discounted');
+
+        if ($discount && result.new_amount_order_discounted) {
+            // Cross module without bridge
+            // Update discount of the order
+            $discount.find('.oe_currency_value').text(result.new_amount_order_discounted);
+
+            // We are in freeshipping, so every carrier is Free
+            $('#delivery_carrier .badge').text(_t('Free'));
```
Comment (nim-odoo) [inline addons/website_sale_delivery/static/src/js/website_sale_delivery.js]: Export translation

Comment (kebeclibre) [reply addons/website_sale_delivery/static/src/js/website_sale_delivery.js]: @nim-odoo 
translation exported,
please approve #enterprise/2651 at the same time

**Comment by kebeclibre** [comment]
robodoo r+

**Pull request closed without merging**